src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -124,10 +125,17 @@
     private final RespondentResponseType respondent2ClaimResponseType;
     private final RespondentResponseType respondent1ClaimResponseTypeToApplicant2;
     private final ResponseDocument respondent1ClaimResponseDocument;
+    private final ResponseDocument respondent1ClaimResponseDocumentCopy;
     private final ResponseDocument respondent2ClaimResponseDocument;
 
     private final YesOrNo applicant1ProceedWithClaim;
+    private final YesOrNo applicant1ProceedWithClaimMultiParty2v1;
```

Comment:
Can we not capture user input in a single field?

Reply:
No as depending on conditions we would need to be different wording in the field which can not be changed programaticaly in ccd
i.e
```

{
  "CaseTypeID": "CIVIL",
  "CaseEventID": "CLAIMANT_RESPONSE",
  "CaseFieldID": "applicant1ProceedWithClaimMultiParty2v1",
  "PageDisplayOrder": 1,
  "PageFieldDisplayOrder": 5,
  "DisplayContext": "MANDATORY",
  "PageID": "RespondentResponse",
  "ShowSummaryChangeOption": "Y",
  "FieldShowCondition": "claimantResponseScenarioFlag=\"TWO_V_ONE\""
},
{
  "CaseTypeID": "CIVIL",
  "CaseEventID": "CLAIMANT_RESPONSE",
  "CaseFieldID": "applicant1ProceedWithClaimAgainstRespondent1MultiParty1v2",
  "PageDisplayOrder": 1,
  "PageFieldDisplayOrder": 5,
  "DisplayContext": "MANDATORY",
  "PageID": "RespondentResponse",
  "ShowSummaryChangeOption": "Y",
  "FieldShowCondition": "claimantResponseScenarioFlag=\"ONE_V_TWO_DIFFERENT_SOLICITOR\" OR claimantResponseScenarioFlag=\"ONE_V_TWO_SAME_SOLICITOR\""
}```

Reply:
```{
  "CaseTypeID": "CIVIL",
  "ID": "applicant1ProceedWithClaimMultiParty2v1",
  "Label": "Does Claimant 1 want to proceed with the claim against ${respondent1.partyName} (need to figure out how to display partyName value of respondent1 2v1)",
  "FieldType": "YesOrNo",
  "SecurityClassification": "Public"
},
{
  "CaseTypeID": "CIVIL",
  "ID": "applicant2ProceedWithClaimMultiParty2v1",
  "Label": "Does Claimant 2 want to proceed with the claim against ${respondent1.partyName} (need to figure out how to display partyName value of respondent2 2v1)",
  "FieldType": "YesOrNo",
  "SecurityClassification": "Public"
},
{
  "CaseTypeID": "CIVIL",
  "ID": "applicant1ProceedWithClaimAgainstRespondent1MultiParty1v2",
  "Label": "Do you want to proceed with the claim against ${respondent1.partyName} (need to figure out how to display partyName value of respondent1 1v2)",
  "FieldType": "YesOrNo",
  "SecurityClassification": "Public"
}```